websites/M/MangaFreak/dist/metadata.json

```diff
@@ -6,10 +6,11 @@
 	},
 	"service": "MangaFreak",
 	"description": {
-		"en": "Read manga and download for free. Mangafreak has the latest chapter of all your favorite manga. Check the latest chapter of popular manga like Naruto, Boruto, One Piece and Bleach. Enjoy reading. Manga Reader Online."
+		"en": "Read manga and download for free. Mangafreak has the latest chapter of all your favorite manga. Check the latest chapter of popular manga like Naruto, Boruto, One Piece and Bleach. Enjoy reading. Manga Reader Online.",
+		"fr": "Lisez des mangas et téléchargez-les gratuitement. Mangafreak a le dernier chapitre de tous vos mangas favoris. Lisez le dernier chapitre de mangas populaires comme Naruto, Boruto, One Piece et Bleach. Profitez de votre lecture. Lecteur de manga en ligne."
 	},
 	"url": "w13.mangafreak.net",
-	"version": "1.0.0",
+	"version": "1.1.0",
```

Comment:
```suggestion
	"version": "1.0.1",
```

Reply:
Resolved in 24ca84dc922fbfccb07bcb5643f0f12a8392f9d8